beetmoverscript/utils.py

```diff
@@ -81,6 +81,13 @@ def is_partner_action(action):
     return action in PARTNER_REPACK_ACTIONS
 
 
+def is_maven_action(action):
+    """Function to return boolean if we're promoting a release as opposed to a
```

Comment:
Hm, I feel like the comment could be enhanced a bit by adding something maven or Geckoview specific like: "we're promoting a release with gecko view components as opposed to a nightly or regular release" or alike. What do you think? Afterall, it's a total separate behavior, even though the behind-the-scenes S3-mechanics are the reused.

Reply:
Oh right, I think I copied this piece of doc and forgot to update it. Done in https://github.com/mozilla-releng/beetmoverscript/pull/163/commits/6a0a75dc31baa88bb7bef9103abe7f4fdba4df23. Thank you for spotting it!